test/help.test.js

```diff
@@ -45,6 +46,22 @@ describe('loopback generator help', function() {
     process.env.SLC_COMMAND = undefined;
   });
 
+  it('print right help message for each generator', function() {
+    process.env.SLC_COMMAND = 'loopback --help';
+    var names = ['acl', 'app', 'boot-script', 'datasource', 'export-api-def',
+    'middleware', 'model', 'property', 'relation', 'remote-method', 'swagger'];
+
+    names.forEach(function(name) {
+      var gen = givenGenerator(name, ['--help']);
+      var helpText = gen.help();
+      var helpFileName = 'loopback_' + name + '_help.txt';
+      var helpFilePath = '../fixtures/' + helpFileName;
+      var output = fs.readFileSync(helpFilePath, 'utf8');
+      assert.equal(output, helpText);
```

Comment:
Now the return value of `gen.help()` is the **globalized** one, all strings in `lib/help.js` are globalized, e.g.: https://github.com/strongloop/generator-loopback/blob/globalize/help-files/lib/help.js#L48

So now we can compare `gen.help()` and the expected text directly.